bloom-housing/bloom
# feat: add MA sign in styling

# Pull Request Template

## Issue Overview

This PR addresses #3758. 


## Description

As part of the Mandated Accounts MVP, this PR applies the updated Figma form styles to the sign in page (https://github.com/bloom-housing/bloom/issues/3758). 

## How Can This Be Tested/Reviewed?

To review this PR:

1. Go to the `/sign-in` page.
2. Compare the form styling for this page with the [Figma](https://www.figma.com/file/ihaZA5QibRRJFyBdARKn55/User-Accounts?node-id=2591%3A4012&mode=dev) and ensure consistency.

Note: This ticket does not include changing the button styling for "Sign in" and "Create an account" or the show/hide password checkbox. Please let me know if you notice any other styling differences that should be accounted for in this ticket.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
